src/core/ledger/testUtils.js

```diff
@@ -26,6 +43,78 @@ export function createUuidMock(): UuidMock {
   }
 
   randomMock.mockImplementation(nextFakeUuid);
+  return {resetFakeUuid, setNextUuid};
+};
+
+export const createDateMock = (): DateMock => {
+  let nextFakeDate = 0;
+
+  function resetFakeDate() {
+    nextFakeDate = 0;
+  }
+  function setFakeDate(ts: number) {
+    // Use this when you want specific timestamps, rather than just
+    // auto-incrementing
+    jest.spyOn(global.Date, "now").mockImplementationOnce(() => ts);
+  }
+  jest.spyOn(global.Date, "now").mockImplementation(() => nextFakeDate++);
+
+  return {setFakeDate, resetFakeDate};
+};
+
+export const createLedgerMock = (
```

Comment:
IMO this isn't really a mock--more like a fixture (`testLedgerFixture`?).

Reply:
also felt sus to me. will change